Modix.Data/Models/Moderation/DeletedMessageSummary.cs

```diff
@@ -106,10 +106,10 @@ public static ICollection<string> SortablePropertyNames
                 Author = entity.Author.Project(GuildUserBrief.FromEntityProjection),
                 Created = entity.BatchId == null
                     ? entity.CreateAction.Created
-                    : entity.Batch.CreateAction.Created,
+                    : entity.Batch!.CreateAction.Created,
```

Comment:
Wow, how does the compiler not recognize the `null`-check here?

Reply:
The null check is on `BatchId`, not `Batch`.

Reply:
Riiiiiiiiiiiiiiight. Could probably just swap that instead.

Reply:
I assumed that it was cheaper to check the BatchId instead of Batch for some reason. I could just convert it though.

Reply:
I mean, in a very very nitpicky sense of "do the least amount of work" yes, checking the Id accomplishes the same logical thing, with one less level of depth in the check. Which is actually why I tend to write such checks this way, because technically one is "better" in the most microscopic sense.

The difference in SQL amounts to checking one side or the other of a `LEFT JOIN` expression. The query plan should be identical, either way.

However, with nullable-reference-types thrown in, I think the equation of "microscopically better" flips to the other side. One way requires a weird nullability annotation, the other does not.